recommender/engine/rl_agent/rl_agent.py

```diff
@@ -70,5 +106,14 @@ def _create_state(self, user: User, search_data: SearchData) -> State:
             state: State containing information about user and search_data
         """
 
-        # TODO: implement
-        pass
+        search_data = copy.deepcopy(search_data)
+        search_data.pop("rating")
```

Comment:
I would just add "rating" to SearchData model - looks better and we can always just ignore it later if needed

Reply:
There was a discussion about adding rating to the SearchData model and the result was: not to add. So we will stay with it.